pages/iam/reference-content/guests-to-members-migration.mdx

```diff
@@ -0,0 +1,86 @@
+---
+meta:
+  title: IAM Guests to Members migration
+  description: Learn how to migrate IAM Guests to Members, including roles and API keys, with Scaleway's IAM introduction
+content:
+  h1: IAM Guests to Members Migration
+  paragraph: This page guides you through the process of migrating IAM Guests to Members, covering key aspects such as roles and API keys, following the introduction of IAM on Scaleway
+tags: iam migration
+categories:
+  - iam
+  - console
+---
+
+This document explains how user management changes with the migration of IAM Guests to Members.
+
+## IAM Users
+
+A user (also known as an IAM user) is a human user in an Organization. Three types currently exist:
+
+- **Owner**: You are the Owner of the [Organization](#organization) that was created with your account.
+- **Guest**: You are a Guest when invited to another Organization of which you are not the Owner.
```

Comment:
Maybe we can add something like.
You are a Guest when invited to another Organization of which you are not the Owner. As a prerequisite, you  mustown your own Organization on the side